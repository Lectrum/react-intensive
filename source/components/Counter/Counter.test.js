import dom from 'react-test-renderer';
import { Counter } from './';

const renderTree = dom.create(<Counter count = { 5 } />).toJSON();

test('counter component should correspond to its snapshot', () => {
    expect(renderTree).toMatchSnapshot();
});
