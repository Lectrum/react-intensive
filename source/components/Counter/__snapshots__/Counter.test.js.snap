// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`counter component should correspond to its snapshot 1`] = `
<section
  className="counter"
>
  Posts count: 
  5
</section>
`;
